src/actions/breakpoints/addBreakpoint.js

```diff
@@ -50,7 +50,8 @@ export default async function addBreakpoint(
     newGeneratedLocation
   );
 
-  const astLocation = await getASTLocation(sourceRecord, newLocation);
+  const symbols = getSymbols(getState, sourceRecord);
```

Comment:
- `getSymbols` seems not to be defined.
- should be `getSymbols(getState(), sourceRecord)` or `getSymbols(state, sourceRecord)` right?